### DISCUSSION TITLE
Interfaces Missing from classification

### DISCUSSION THREAD

**Discussion by cmurrayis** [akvorado/akvorado#141] [Q&A]
Hi,

We've gone through and classified all interfaces locally on our routers with the Transit/Peering/Core etc. per our regular naming conventions which matched the default "Classify" statements in the Akvorado.yml file:

```
    interface-classifiers:
      - |
        ClassifyConnectivityRegex(Interface.Description, "^(?i)(transit|pni|ppni|peering|core):? ", "$1") &&
        ClassifyProviderRegex(Interface.Description, "^[^ ]+? ([^ ]+)", "$1") &&
        ClassifyExternal()
      - ClassifyInternal()
```
We have also added the SNMP `cache-refresh: 30m` option and restarted docker multiple times however one of our transit interfaces still shows up blank.

Junos MX204

```
xe-0/1/0 {
    description "EdgeIX [10000mbit]";
    per-unit-scheduler;
    vlan-tagging;
    mtu 9000;
    unit 111 {
        description "Peering: EdgeIX QLD";
        vlan-id 111;
        family inet {
            mtu 1580;
            inactive: tcp-mss 1200;
            sampling {
                input;
                output;
            }
            address X.X.X.X/24;
        }
    }
    unit 153 {
        description "Peering: EdgeIX NSW";
        vlan-id 153;
        family inet {
            mtu 1580;
            sampling {
                input;
                output;
            }
            address X.X.X.X/23;
        }
    }
}
```
Yet when looking at traffic and removing the `InIfBoundary = external` filter to show all data the following is blank:

![image](https://user-images.githubusercontent.com/89428026/190270728-3d9028a2-5cd5-48e1-a1fd-c3a92da695be.png)

How would we best troubleshoot this as the other interfaces are described similar in Junos and show up correctly. From the above example you can see 1 of the 2 interfaces appears and I can confirm both have traffic active at the review window.

![image](https://user-images.githubusercontent.com/89428026/190271222-b51ee80a-660a-42f4-b9d7-d94c37be38bd.png)

![image](https://user-images.githubusercontent.com/89428026/190271264-37f739f9-d88b-4582-8154-e4d93d794e67.png)

**Comment by vincentbernat** [comment]
You could use `tcpdump` to spy the SNMP requests that Akvorado is doing to the problematic device and check if the request for xe-0/1/0.153 is executed and if it's the right SNMP index.
